cli/src/core/deploy/deploy.go

```diff
@@ -0,0 +1,127 @@
+package deploy
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"regexp"
+	"strconv"
+
+	"cli/core"
+	"cli/core/parse"
+	"cli/util/file"
+	"cli/util/git"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/luno/jettison/errors"
+	cp "github.com/otiai10/copy"
+)
+
+func MountCustomPackage(ctx context.Context, cli *client.Client, customPackage core.CustomPackage, instantContainerId string) error {
```

Comment:
I think it is better to split this function into small function each one for a type, 
for more readability. What do you think ?